TurkishFilter/sections/specific.txt

```diff
@@ -86,6 +86,7 @@ sineturk.pw,altyazilifilmizle.tv,1080pfilmizle.net,dolufilmizle.com,filmizleve.c
 ||filmkolikhd.net/wp-content/uploads/*/pageskinnnnn.jpg
 /970x250-$image,domain=kriptokoin.com
 /Lunabit.gif$domain=kriptokoin.com
+$third-party,image,domain=iyinet.com
```

Comment:
bad rule. This is forum, users can upload images from any source.
Better use something like that:
```
iyinet.com##.p-body-inner > center > a[title]:not([href^="https://iyinet.com/"]) > img[alt]
iyinet.com##.p-header-content > a[title]:not([href^="https://iyinet.com/"]) > img[alt]
```